modules/lang-painless/src/main/java/org/elasticsearch/painless/action/PainlessExecuteAction.java

```diff
@@ -552,8 +552,10 @@ private static Response prepareRamIndex(Request request,
                     indexWriter.addDocuments(parsedDocument.docs());
                     try (IndexReader indexReader = DirectoryReader.open(indexWriter)) {
                         final long absoluteStartMillis = System.currentTimeMillis();
+                        final IndexSearcher searcher = new IndexSearcher(indexReader);
+                        searcher.setQueryCache(null);
```

Comment:
just a small question on this. where did we do setQueryCache(null) before?

Reply:
It shouldn't be required, I added this by precaution but I can remove it

Reply:
I see, it's fine by me, I was just curious

Reply:
and what about the places where we were previously setting the query cache and the policy?